api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,192 @@
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (13, 13, 1);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (14, 1, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (15, 2, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (16, 3, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (17, 4, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (18, 5, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (19, 6, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (20, 7, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (21, 8, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (22, 9, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (23, 10, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (24, 11, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (25, 12, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (26, 13, 2);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (27, 1, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (28, 2, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (29, 3, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (30, 4, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (31, 5, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (32, 6, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (33, 7, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (34, 8, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (35, 9, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (36, 10, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (37, 11, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (38, 12, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (39, 13, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (40, 1, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (41, 2, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (42, 3, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (43, 4, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (44, 5, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (45, 6, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (46, 7, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (47, 8, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (48, 9, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (49, 10, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (50, 11, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (51, 12, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (52, 13, 4);
```

Comment:
You may want to combine all of these `INSERT` statements into a single `INSERT ALL` statement https://www.techonthenet.com/oracle/questions/insert_rows.php

Reply:
It's 52 lines of code I generated using Python and insert all does not greatly reduce the code amount so I don't think I need to change it

Reply:
The main benefit of using an `INSERT ALL` statement would be to use 1 database query instead of 52 database queries to insert the rows. This wouldn't reduce the code size, but would reduce insertion time and database traffic. This isn't necessary though, just something to keep in mind.